.gitignore

```diff
@@ -8,6 +8,7 @@ __pycache__/
 
 # Distribution / packaging
 .Python
+
```

Comment:
Why did you add the line here?

Reply:
More precisely, I think you should remove this since there is no point in this change.
```suggestion
```